src/hooks/useReportScrollManager/index.native.ts

```diff
@@ -1,27 +1,30 @@
 import {useContext, useCallback} from 'react';
-import {ActionListContext} from '../../pages/home/ReportScreenContext';
+import {ActionListContext, ActionListContextType} from '../../pages/home/ReportScreenContext';
 
-function useReportScrollManager() {
+function useReportScrollManager(): {
+    ref: ActionListContextType;
+    scrollToIndex: (index: number) => void;
+    scrollToBottom: () => void;
+} {
     const flatListRef = useContext(ActionListContext);
 
     /**
      * Scroll to the provided index.
      *
-     * @param {Object} index
      */
-    const scrollToIndex = (index) => {
-        if (!flatListRef.current) {
+    const scrollToIndex = (index: number) => {
+        if (!flatListRef?.current) {
             return;
         }
 
-        flatListRef.current.scrollToIndex(index);
+        flatListRef.current.scrollToIndex({index});
```

Comment:
why this change? 
If its necessary please make sure its not causing any regressions

Reply:
It's correct now - https://reactnative.dev/docs/flatlist#scrolltoindex

Reply:
⬆️ exactly 😄

Reply:
But it wasn't correct before, and it can cause a regression. Please test it 🙌